core/lib/dr_tools.h

```diff
@@ -341,6 +341,12 @@ typedef enum {
      * \note Windows only.
      */
     DR_MEMORY_DUMP_LDMP = 0x0001,
+    /**
+     * Memory dump in Executable and Linkable Format.
+     *
+     * \note Linux and X64 only.
```

Comment:
Add a release note to api/docs/release.dox on the new feature.

Reply:
I've updated the doc to put it under "The changes between version \DR_VERSION and 11.0.0 include the following compatibility".